src/System.Data.Common/tests/System/Data/Common/RowUpdatedEventArgsTest.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Xunit;
+
+namespace System.Data.Common
+{
+    public class RowUpdatedEventArgsTest
+    {
+        [Fact]
+        public void ConstructorFail()
+        {
+            ArgumentOutOfRangeException ex = Assert.Throws<ArgumentOutOfRangeException>(() => new RowUpdatedEventArgs(null, null, (StatementType)100, null));
+            Assert.Equal(typeof(ArgumentOutOfRangeException), ex.GetType());
+            Assert.Null(ex.InnerException);
+            Assert.True(ex.Message.IndexOf(nameof(StatementType)) != -1);
+            Assert.True(ex.Message.IndexOf("100") != -1);
+            Assert.Equal(ex.ParamName, nameof(StatementType));
+        }
+
+        [Fact]
+        public void ConstructorPass()
+        {
+            var ev = new RowUpdatedEventArgs(null, null, 0, null);
+            Assert.NotNull(ev);
+            Assert.Null(ev.Row);
+            Assert.Null(ev.Command);
+            Assert.Null(ev.TableMapping);
+            Assert.Equal(ev.StatementType, (StatementType)0);
+            Assert.Equal(ev.RowCount, 0);
+            Assert.Equal(ev.RecordsAffected, 0);
```

Comment:
All of these Equal calls should have their arguments reversed; same for other cases below.  The Equal method takes arguments `expected` and `actual`, meaning the first argument should be the expected result and the second argument should be the thing we're verifying.